lib/powerpoint/version.rb

```diff
@@ -1,3 +1,3 @@
 module Powerpoint
-  VERSION = "1.7.3"
+  VERSION = "1.7.4"
```

Comment:
You're adding an entire feature here and not patching a bug, so I'd bump this a minor.